src/parameter_input.cpp

```diff
@@ -848,87 +848,6 @@ std::string ParameterInput::SetString(const std::string &block, const std::strin
   return value;
 }
 
-//----------------------------------------------------------------------------------------
-//! \fn void ParameterInput::RollbackNextTime()
-//  \brief rollback next_time by dt for each output block
-
-void ParameterInput::RollbackNextTime() {
-  InputBlock *pb = pfirst_block;
-  InputLine *pl;
-  std::stringstream msg;
-  Real next_time;
-
-  while (pb != nullptr) {
-    if (pb->block_name.compare(0, 16, "parthenon/output") == 0) {
-      pl = pb->GetPtrToLine("next_time");
-      if (pl == nullptr) {
-        msg << "### FATAL ERROR in function [ParameterInput::RollbackNextTime]"
-            << std::endl
-            << "Parameter name 'next_time' not found in block '" << pb->block_name << "'";
-        PARTHENON_FAIL(msg);
-      }
-      next_time = static_cast<Real>(atof(pl->param_value.c_str()));
-      pl = pb->GetPtrToLine("dt");
-      if (pl == nullptr) {
-        msg << "### FATAL ERROR in function [ParameterInput::RollbackNextTime]"
-            << std::endl
-            << "Parameter name 'dt' not found in block '" << pb->block_name << "'";
-        PARTHENON_FAIL(msg);
-      }
-      next_time -= static_cast<Real>(atof(pl->param_value.c_str()));
-      msg << next_time;
-      // AddParameter(pb, "next_time", msg.str().c_str(), "# Updated during run time");
-      SetReal(pb->block_name, "next_time", next_time);
-    }
-    pb = pb->pnext;
-  }
```

Comment:
Why is this being removed in this PR?

Reply:
Yeah I think that's right. I deleted them because they're completely unused.

As an aside, something I want to add to another MR maybe is documentation for how to change output cadence upon restart from the command line. This is totally possible to do, but requires setting both `parthenon/output/dt` and `parthenon/output/next_time`.